### PR TITLE
D8CORE-3352: updated configurations for past events

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- With these config changes, we remove the `stanford_events_past` view, and consolidate it into the `stanford_events` view, similar to what we did with the schedule view.  
- We update the `events/past` page to use the new view display.
- We add/change a few things to match the designs in figma (https://www.figma.com/file/LSfhdmKiOlNniVMvFkvigw/SWS-D8-Events-and-Event-Series-WF-%2B-Mocks?node-id=1823%3A13218)
- Accompanying style changes are in the events module, with this pr: https://github.com/SU-SWS/stanford_events/pull/58

# Needed By (Date)
- 2.0.1 code cutoff

# Urgency
- medium

# Steps to Test

1. checkout this branch.
2. checkout the corresponding branch in the `stanford_events` module (see above)
3. `drush cim`
4. `drush cr`
5. Create some events that occur in the past.  navigate to `events/past` and verify the page looks right, per the Figma mocks (see above)

# Affected Projects or Products
- `stanford_events`: https://github.com/SU-SWS/stanford_events/pull/58

# Associated Issues and/or People
- D8CORE-3352
- See above.
- I'd like to demo this for @ishahism to make sure it meets her approval.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by imonroe** [comment]
Went over this visually with Ishi, and she's good with it.

**Review by pookmish — COMMENTED** [review]
With a quick review, this seems fine. But because that view is being deleted, we'll need to do a database update to replace any blocks in one-off layout builder nodes. 
Also if you want the blocks to be added via the list paragraph type, you'll need to switch the desired ones from `block` to `viewfield_block` like [this line](https://github.com/SU-SWS/stanford_profile/pull/372/files#diff-e9bc24256c0d77b6c3ed564bdfdc23d4ae9c692bf1b66327e8e58af9d35723afR4721)

**Comment by imonroe** [comment]
Should I just not delete the existing view that we're replacing?  That means we don't need to do a DB update.  We also still have the old Schedule view in there, even though we replaced it with a display in the events view as well.  Maybe we leave both alone and then do the removals and DB updates at the same time as part of a different PR?

**Comment by pookmish** [comment]
Yeah I was thinking the same thing. lets leave it and create a cleanup ticket in Jira to do the removal and db updates.

**Comment by pookmish** [comment]
@imonroe  Any update on this to get into todays code freeze?

**Comment by imonroe** [comment]
yeah, working on it now.

File: config/sync/page_manager.page.stanford_events_past.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-uuid: daa79d68-e5bc-49ec-a8bc-799d2eba17fc
+uuid: 18949aa4-325b-44fe-b860-0ca64500feb0
```
Comment (pookmish) [inline config/sync/page_manager.page.stanford_events_past.yml:1]: UUID's should never change

File: config/sync/page_manager.page_variant.stanford_events_past-layout_builder-0.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-uuid: 53d33dc2-9ed4-462c-9c8a-641c026651c5
+uuid: 324b692f-6be0-407c-a6db-1dff67da7be1
```
Comment (pookmish) [inline config/sync/page_manager.page_variant.stanford_events_past-layout_builder-0.yml:1]: UUIDs should never change

**Comment by imonroe** [comment]
I rerolled this to get rid of the messy commit history.  reroll: https://github.com/SU-SWS/stanford_profile/pull/378

I'm going to close this PR in favor of that one.

**Pull request closed without merging**